pkg/asset/tls/apiserver.go

```diff
@@ -31,27 +31,13 @@ func (a *APIServerCertKey) Dependencies() []asset.Asset {
 // Generate generates the cert/key pair based on its dependencies.
 func (a *APIServerCertKey) Generate(dependencies asset.Parents) error {
 	kubeCA := &KubeCA{}
-	installConfig := &installconfig.InstallConfig{}
```

Comment:
you should also drop the dependency from https://github.com/openshift/installer/pull/1640/files#diff-c7316d3d0bee102140cf53dc3ae5edf9R27